torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -26,50 +27,12 @@
 from torch.utils import _pytree
 
 
-def _onnx_function_diagnose_call_message_formatter(
-    fn: Callable, args: Tuple[Any, ...], kwargs: Dict[str, Any]
-) -> str:
-    if len(args) > 0 and isinstance(args[0], onnxscript.OnnxFunction):
-        onnx_function: onnxscript.OnnxFunction = args[0]  # self
-        return f"{onnx_function.name}: {onnxscript.OnnxFunction}"
-    return f"{fn.__name__}: {fn}"
-
-
-def _onnx_function_diagnose_call_append_symbolic_source_location(
-    diagnostic: diagnostics.infra.Diagnostic,
-    fn: Callable,
-    args: Tuple[Any, ...],
-    kwargs: Dict[str, Any],
-    return_values: Any,
-) -> None:
-    # TODO(bowbao): Record source location of symbolic.
-    # Need this separate step because normally only the source location of
-    # class `onnxscript.OnnxFunction.__call__` is recorded.
-    ...
-
-
-# TODO(bowbao): Delete this once diagnostics is introduced in onnxscript.
-_diagnose_onnx_function = diagnostics.diagnose_call(
-    rule=diagnostics.rules.atenlib_symbolic_function,
-    diagnostic_message_formatter=_onnx_function_diagnose_call_message_formatter,
-    diagnostic_modifier=_onnx_function_diagnose_call_append_symbolic_source_location,
-)
-for key, onnx_function in function_dispatcher._ATENLIB_FUNCTIONS.items():
-    if isinstance(onnx_function, FunctionType):
-        function_dispatcher._ATENLIB_FUNCTIONS[key] = _diagnose_onnx_function(
-            onnx_function
-        )
-onnxscript.OnnxFunction.__call__ = _diagnose_onnx_function(
-    onnxscript.OnnxFunction.__call__
-)
-
-
 @_beartype.beartype
 def _fx_node_to_onnx_message_formatter(
     fn: Callable, args: Tuple[Any, ...], kwargs: Dict[str, Any]
 ) -> str:
-    assert len(args) > 0
-    node = args[0]
```

Comment:
Comment on 1 because it seems tricky?

Reply:
Sure. I was thinking in another PR to change the signature style along with the infra to 
```
def _fx_node_to_onnx_message_formatter(
    fn: Callable, diagnostic_context: DiagnosticContext, node: torch.fx.Node, *args, **kwargs
) -> str:
```
wdyt?

Reply:
Looking good!